client/libs/react-app-context.jsx

```diff
@@ -3,12 +3,16 @@ import {React} from 'meteor/react-runtime';
 
 export function applyContext(context, _actions) {
   const actions = {};
-  for (var key in _actions) {
-    const actionMap = _actions[key];
-    for (var actionName in actionMap) {
-      actionMap[actionName] = actionMap[actionName].bind(null, context);
+  for (let key in _actions) {
```

Comment:
My intuition was to use ES6 for ... of here, but unfortunately vanilla JS object are non-iterable, so you may consider switching to an iterable data structure (such as an Array, a Map, or a Set).